go-controller/pkg/allocator/ip/allocator.go

```diff
@@ -40,7 +40,7 @@ type Interface interface {
 }
 
 var (
-	ErrFull      = errors.New("range is full")
+	ErrFull      = errors.New("range is used up")
```

Comment:
nit: I think the word "exhausted" would be better. Maybe "subnet address pool exhausted"